test/ModuleInterface/full-convention.swift

```diff
@@ -0,0 +1,32 @@
+// RUN: %target-swift-frontend -typecheck -swift-version 5 -emit-module-interface-path - -enable-library-evolution %s -experimental-print-full-convention | %FileCheck %s
+
+public func f(
+  // CHECK: g: @convention(c, cType: "void (void)")
+  g: @convention(c) () -> (),
```

Comment:
Is this somehow not the normal colon character?  I don't know why GitHub is being weird about it.

Reply:
I don't think I did anything funny, but I will double-check.

Reply:
I suspect GitHub's Swift parser is buggy. You'll notice that it is syntax-highlighting the `CHECK` and the `@convention` and the string in the comments, which doesn't make much sense IMO.

Reply:
The red colon is probably something like "I didn't expect a colon here, you probably have a syntax error".